client/cmd.go

```diff
@@ -111,6 +111,11 @@ func ReadPersistentCommandFlags(clientCtx Context, flagSet *pflag.FlagSet) (Cont
 		clientCtx = clientCtx.WithKeyringDir(keyringDir)
 	}
 
+	if !clientCtx.DryRun || flagSet.Changed(flags.FlagDryRun) {
```

Comment:
Please try reuse the `--simulate` flag.

Reply:
It seems `--simulate` is only ever used for tx commands, I figured it makes more sense sticking to the `--dry-run` flag and not break UX.

I can assign `--dry-run` value to `--simulate` and remove an additional context variable alternatively.

Reply:
> I can assign --dry-run value to --simulate and remove an additional context variable alternatively.

I like that :+1:

Reply:
Is agree with sticking with --simulate to reduce confusion

Reply:
As far as I understand query/keys commands use `dry-run` while tx uses `simulate`. Since this association was already wired and ready I figured I would move said wiring in a more general place, so now it's used for query/keys commands too.